pkg/handler/repositories.go

```diff
@@ -97,10 +126,67 @@ func (rh *RepositoryHandler) createRepository(c echo.Context) error {
 		return echo.NewHTTPError(httpCodeForError(err), "Error creating repository: "+err.Error())
 	}
 
+	if err = rh.produceMessage(c, response); err != nil {
+		log.Logger.Warn().Msgf("error producing kafka message: %s", err.Error())
+	}
+
 	c.Response().Header().Set("Location", "/api/"+config.DefaultAppName+"/v1.0/repositories/"+response.UUID)
 	return c.JSON(http.StatusCreated, response)
 }
 
+func (rh *RepositoryHandler) produceMessage(c echo.Context, response api.RepositoryResponse) error {
+	var (
+		msg       message.IntrospectRequestMessage
+		headerKey string
+		err       error
+	)
+	// Fill Key
+	key := response.UUID
+
+	// Read header values
+	headerKey = string(message.HdrXRhIdentity)
+	xrhIdentity := GetHeader(c, headerKey, []string{})
+	if len(xrhIdentity) == 0 {
+		return fmt.Errorf("expected a value for '%s' http header", headerKey)
+	}
+	headerKey = string(message.HdrXRhInsightsRequestId)
+	xrhInsightsRequestId := GetHeader(c, headerKey, []string{})
+	if len(xrhInsightsRequestId) == 0 {
+		return fmt.Errorf("expected a value for '%s' http header", headerKey)
+	}
+
+	// Fill headers
+	headers := []kafka.Header{
+		{
+			Key:   string(message.HdrType),
+			Value: []byte(message.HdrTypeIntrospect),
+		},
+		{
+			Key:   string(message.HdrXRhIdentity),
+			Value: []byte(xrhIdentity[0]),
+		},
+		{
+			Key:   string(message.HdrXRhInsightsRequestId),
+			Value: []byte(xrhInsightsRequestId[0]),
+		},
+	}
+
+	// Fill topic
+	topic := schema.TopicIntrospect
```

Comment:
BUG! this should use the real topic name; add a mapping based into clowder configuration; by default the mapping will match the same topic so it is easier to deal into the developing environment (local workstation and ephemeral environment).